013-move-docker-images-to-quay.io.md

```diff
@@ -0,0 +1,43 @@
+# Move Container Images to Quay.io
+
+Docker is introducing from November 2020 service limits for container images stored in Docker Hub.
+Free accounts will have following rate limits:
+* 100 pulls for anonymous users per 6 hours
+* 200 pulls for authenticated users (on the free plan) per 6 hours
+
+In addition, images will be kept only for 6 months.
```

Comment:
I don't think this is correct. I think it's only when an image tag has not been pulled or pushed for 6 months that it will be deleted. So long as someone (potentially) pulled images every few months the images would remain indefinitely.

Reply:
I guess that is right, but I'm not sure this makes it any better - just less predictable.